### PR TITLE
documentation updates for EasyBuild v2.1.0

### PR DESCRIPTION
**Pull request by boegel**
links to the rendered version of the updated page are available via the changlog entry for the `20150430.01` docs release @ http://boegel-eb.readthedocs.org/en/eb210/Changelog.html

@wpoely86: please review

### CODE REVIEW

File: docs/Manipulating_dependencies.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+.. _filter_deps:
+
+Filtering out dependencies using ``--filter-deps``
+--------------------------------------------------
+
+To avoid that certain dependencies are being installed, a list of software names can be specified to ``--filter-deps``.
+Any time a dependency with a name from this list is specified, it will be simply filtered out by EasyBuild, and
+thus disregarded when resolving dependencies, loading modules for the dependencies in the build environment, and
+including '``module load``' statements in the generated module files.
```
Comment (wpoely86) [inline docs/Manipulating_dependencies.rst]: the whole thing is version agnostic?

File: docs/Partial_installations.rst [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+For this purpose, the ``-k``/``--skip`` command line option is available.
+
+To actually skip an existing software installation and all installed extensions, a corresponding module must be
+available already; if not, the installation procedure will be performed from scratch.
+To trigger the installation of missing extensions, ``--force`` must be used as well; without it, the installation
+procedure will be skipped as a whole (since the module is already available).
+
+When ``--skip`` is combined with ``--force``, EasyBuild will:
+
+i) ensure that all (extension) sources are available (and try to fetch them if needed);
```
Comment (wpoely86) [inline docs/Partial_installations.rst]: maybe add that it does not work for upgrading extensions?

File: docs/Partial_installations.rst [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+   $ eb GCC-4.8.2.eb --module-only --force
+   ...
+   == building and installing GCC/4.8.2...
+   ...
+   == sanity checking [skipped]
+   == creating module...
+   == COMPLETED: Installation ended successfully
+   ...
+
+   $ ls -l /home/example/.local/modules/all/GCC/4.8.2*
```
Comment (wpoely86) [inline docs/Partial_installations.rst]: why the star?

File: docs/Manipulating_dependencies.rst (hunk 1 above)
Comment (boegel) [inline docs/Manipulating_dependencies.rst]: for now, yes, patches welcome ;)

File: docs/Partial_installations.rst (hunk 2 above)
Comment (boegel) [inline docs/Partial_installations.rst]: makes sense, will do

File: docs/Partial_installations.rst (hunk 3 above)
Comment (boegel) [inline docs/Partial_installations.rst]: copy-paste from the `.lua` example, will fix, thx

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**